mcs/class/referencesource/mscorlib/system/security/cryptography/rc2cryptoserviceprovider.cs

```diff
@@ -63,7 +63,12 @@ public sealed class RC2CryptoServiceProvider : RC2 {
                 return m_use40bitSalt;
             }
             set {
+#if MONO
```

Comment:
I meant to move the check to CreateEncryptor/CreateDecryptor and test only true value